src/components/auth/RegisterChildren.vue

```diff
@@ -545,6 +554,7 @@ function codeNotRight(index) {
 function deleteStudentForm(student) {
   if (state.students.length > 1) {
     state.students.splice(student, 1); // Remove the student at the specified index
+    showConsent.value.splice(student, 1);
```

Comment:
```suggestion
function deleteStudentForm(student) {
  if (state.students.length > 1) {
    state.students.splice(student, 1); // Remove the student at the specified index
    showConsent.value.splice(student, 1);
```
Did you intend to leave the function parameter as `student` rather than `index`?

Either way is fine; just double checking.